### ISSUE TITLE
TwentyTwenty: keyboard focus not set on Close button in main menu

### ISSUE THREAD

**Issue by schlessera** [ampproject/amp-wp#3618]
## Bug Description

From https://github.com/ampproject/amp-wp/pull/3606#issuecomment-545398633:

In TwentyTwenty, the keyboard focus is not set to the Close button when opening the main menu.

## Expected Behaviour

When opening the main menu in TwentyTwenty, the Close button should have keyboard focus.

## Steps to reproduce

1. Install and activate TwentyTwenty
2. Open an AMP URL.
3. Open the main menu in mobile view.
4. The Close button should have keyboard focus but doesn't.

## Screenshots

AMP version:
![image](https://user-images.githubusercontent.com/134745/67388408-6b1f2180-f54d-11e9-9038-f88d9d936c67.png)

Non-AMP version:
![image](https://user-images.githubusercontent.com/134745/67388429-770ae380-f54d-11e9-985c-be9c458c11ac.png)

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

**Comment by csossi** [comment]
Verified in QA

AMP-version:
![image](https://user-images.githubusercontent.com/16945664/67731079-60411280-f9cd-11e9-8e38-eea41a9f5b51.png)
